action.yml

```diff
@@ -24,6 +24,12 @@ inputs:
       See https://nixos.org/manual/nix/stable/command-ref/conf-file.html for
       information on what settings that are available. Make sure the settings
       you define are supported by the Nix version you're using.
```

Comment:
Can't we extract the experimental-features from the nix_conf setting instead? We could use grep and cut to append it.

Setting this into an extra option feels a bit hacky.